src/NuGet.Core/NuGet.Packaging/PackageCreation/Authoring/PhysicalPackageFile.cs

```diff
@@ -47,7 +47,8 @@ public string TargetPath
             {
                 if (String.Compare(_targetPath, value, StringComparison.OrdinalIgnoreCase) != 0)
                 {
-                    _targetPath = value;
+                    // Always use forward slash in package.
+                    _targetPath = value.Replace("\\", "/");
```

Comment:
this doesn't seem like it's right. what if the user actually meant "\" to be a part of his targetPath on a unix system?